tests/scripts/docker/run_tests.sh

```diff
@@ -25,6 +25,6 @@ docker run \
   /bin/bash \
   -c "sudo service docker start
       pip install --user -U pip
-      pip install --user -r requirements-tests.txt --cache-dir ${docker_pip_cache}
       pip install --user -r requirements.txt --cache-dir ${docker_pip_cache}
+      pip install --user -r requirements-tests.txt --cache-dir ${docker_pip_cache}
```

Comment:
`pip install --user -r requirements.txt --cache-dir ${docker_pip_cache}`

having separated requirements file according to the scope, only executing the only one used for building the lambda we assure the exact intended versions dependencies are installed

if we change the order potentially an indirect dependency from the other two requirements file could change the final versions installed

there's probably a better way to insure this but I don't know about it